build/install_windows.ps1

```diff
@@ -0,0 +1,8 @@
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+$PSDefaultParameterValues['*:ErrorAction']='Stop'
+
+Set-Location "$Env:TRAVIS_BUILD_DIR"
+git clone "https://github.com/plus3it/amazon-inspector-formula"
+$saltDir = "C:\salt\srv\salt\"
+Copy-Item "$Env:TRAVIS_BUILD_DIR\amazon-inspector-formula\" -Destination $saltDir -Recurse
```

Comment:
Not sure on `Copy-Item` semantics, but same idea, may need to adjust the copy path